.appveyor.yml

```diff
@@ -15,15 +15,15 @@ environment:
   OPENPACE_VER: 1.1.3
   ZLIB_VER_DOT: 1.3.1
   matrix:
-  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-    VCVARSALL: "%ProgramFiles(x86)%/Microsoft Visual Studio/2017/Community/VC/Auxiliary/Build/vcvarsall.bat"
-    DO_PUSH_ARTIFACT: yes
-    OPENSSL_VER: "1.1.1"
-    OPENSSL_DIR: "C:\\OpenSSL-v111"
+  #- APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+  #  VCVARSALL: "%ProgramFiles(x86)%/Microsoft Visual Studio/2017/Community/VC/Auxiliary/Build/vcvarsall.bat"
+  #  OPENSSL_VER: "1.1.1"
+  #  OPENSSL_DIR: "C:\\OpenSSL-v111"
```

Comment:
If it doesn't hurt, then I would still include VS2017 in CI. (upload artifact is OK to be done from VS 2019)

Reply:
I will try, the dotnet was old to execute wix process. Maybe I can figure out some workaround

Reply:
if it would be to complicated, there is no problem in dropping VS 2017

Reply:
Fixed